.github/workflows/ruby-ci.yml

```diff
@@ -25,12 +25,16 @@ jobs:
           - gemfiles/Gemfile.latest-release
           - gemfiles/Gemfile.shopify
           - gemfiles/Gemfile.rails-edge
-        # exclude:
-        #   - version: 2.7
-        #     gemfile: <Gemfile> to exclude for Ruby Version above
+         exclude:
+           - version: 2.5
+               gemfile:
```

Comment:
- The indent is off, does it matter?
- What does it mean to exclude a gemfile fro a specific ruby version? Does it mean the version is skipped altogether? If yes, should the version be removed on line 20.
   Or is the gem optional? 
   
   
Tx Douglas!!

Reply:
- Yup, I was just fixing the indentation =)
- About excluding a Gemfile it means that we define a set of Gemfiles supported by the application in this part:

```
gemfile:
    - gemfiles/Gemfile.rails60
    - gemfiles/Gemfile.latest-release
    - gemfiles/Gemfile.shopify
    - gemfiles/Gemfile.rails-edge
```

But as we know, some Ruby versions don't work with some of these Gemfiles, so we use the exclude block to remove the Gemfile (and the test run) for the problematic Ruby/Gemfile.